tools/project_manager/show_word_context.php

```diff
@@ -112,7 +126,7 @@
     mysqli_free_result($pages_res);
 
     if($foundInstances>=$wordInstances)
-        echo "<p>" . _("More instances were found, stopping after a small sample.") . "</p>";
+        echo "<p>" . _("More instances were found; please choose how many to show from the drop-down, up to 100.") . "</p>";
```

Comment:
I would remove `, up to 100` so if we change the max size later we don't have to have the string retranslated. You could also just `sprintf()` the string but I don't think it's important to include the number here.

Reply:
It just occurred to me, should the max size be a named constant, anyway, or is that irrelevant here?

Reply:
I think it's worth making a constant since we use it in `get_integer_param()` and in the `range()`. I still think removing it from here is fine.

Reply:
Removed from string and used constant for range and get_integer_param.